classes/local/util/manager.php

```diff
@@ -748,7 +781,7 @@ protected function prepare_lang_file() {
         $this->init_file('lang/en/'.$this->recipe['component'].'.php', 'lang', [
```

Comment:
The prepare_lang_file() function will be deleted in a future patch, it isn't used anywhere.